modules/smilewantedBidAdapter.js

```diff
@@ -1,6 +1,7 @@
 import { isArray, logError, logWarn, isFn, isPlainObject } from '../src/utils.js';
 import { Renderer } from '../src/Renderer.js';
 import { config } from '../src/config.js';
+import { createEidsArray } from './userId/eids.js';
```

Comment:
please don't import from other modules, the eids are available on the bid request object